### ISSUE TITLE
Feedback from #spookycode workshop

### ISSUE THREAD

**Issue by Kriesse** [cssconf/CSSclasses#16]
#### Please collect your suggestions, observations, ideas :ghost:

**Comment by lukaszklis** [comment]
One more suggestion from one of the attendees:
> Provide screenshots for each of the steps in the tutorial, so the attendee would know what is the expected result

**Comment by stefanjudis** [comment]
I haven't checked the material in detail again, but in my class a lot of people were struggling with HTML elements.

Questions like:
- "what is an id?"
- "what is a class?"
...

appeared again and again.

A few additions there might help. 
Can check it and provide PR next week. :)

**Comment by Kriesse** [comment]
### Organization:
* create an intro presentation to kick off the workshop:
(Welcome, who are we, Code of Conduct, Schedule, What is CSS)
* remind people to install Sublime before the workshop starts
* if we work in groups, give out numbers at registration, not later.

### Learning Materials:
* Simplify the HTML part. Beginners were confused by attributes. We should explain classes and IDs later in the second part.
* I propose to create an alternative version of the learning materials. The long-read version we currently have works great for self-directed learning (e.g. in a typical small event at co.up).
For events where we teach beginners in small groups, a slidedeck would be very helpful. C.f. the one from Clojurebridge:
http://clojurebridge.github.io/curriculum/#/1

**Comment by salsab1l** [comment]
I found the workshop great - thanks to all the coaches and organizers, thank you for your time and effort. I went through the CSS classes tutorial at my own pace (which I found great!) and whenever I struggled, I asked a coach for some help. So I would not like to listen to a long introduction and/or a detailed explanation before actually starting to DO something. The great thing in HTML and CSS for me was that I put some commands and directly saw the results. So this way of learning by doing (and seeing) was perfectly fine for me. Again: big thanks to everyone!
